app/javascript/utils/form/final_form/Field.js

```diff
@@ -10,7 +10,12 @@ const {
 
 // Make react-final-form does include empty values on submit.
 // Source: https://github.com/final-form/react-final-form/issues/130#issuecomment-425482365
-const identity = value => (value)
+const identity = (value, name, isCurrency = false) => {
+  if (value && isCurrency) {
+    return value.toString().replace(/[^\d|.]/g, '')
```

Comment:
can you clarify what this is doing here? I think it's worth adding a comment too

Reply:
added a comment